Formula/wdt.rb

```diff
@@ -0,0 +1,50 @@
+class Wdt < Formula
+  desc "Transfer data as fast as possible over multiple TCP paths"
+  homepage "https://www.facebook.com/WdtOpenSource"
+  url "https://github.com/facebook/wdt.git", :revision => "bc22626deede683d11a2668121ebbd2d0389a7b8"
+  version "1.27.1612021-145-gbc22626"
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on "double-conversion"
+  depends_on "folly"
+  depends_on "gflags"
+  depends_on "glog"
+  depends_on "openssl"
+
+  resource "folly" do
```

Comment:
We already have a `folly` formula, please depend on that.

Reply:
It requires cpp files from Folly to build, which is why it requires folly formula (for headers) and also resources folly (for headers and cpps during build time)

Reply:
I think maybe something could be changed upstream to link against a full version of folly rather than building libfolly4wdt, which seems to be a subset of the whole lib. I made an issue here https://github.com/facebook/wdt/issues/194

Reply:
I have added a patch here to cover building against full folly and removing the resource. I am going to try to get something similar upstream.